spec/features/home_spec.rb

```diff
@@ -93,14 +103,53 @@ def login_user
 
     it "selects only relevant cards to display" do
       @test_card = FactoryGirl.create :card, review_date: Date.tomorrow,
+                                             deck_id: deck.id,
                                              user_id: user.id
       expect(page).to have_content "Нет карточек для повторения"
     end
 
     it "Doesn`t show cards of other user" do
       @test_card = FactoryGirl.create :card, review_date: Date.today,
+                                             deck_id: deck.id,
                                              user_id: second_user.id
       expect(page).to have_content "Нет карточек для повторения"
     end
   end
+
+  describe "card selection process with current deck chosen" do
+    let(:alter_deck) { FactoryGirl.create :deck, title: "alter_title",
+                                                 user_id: user.id 
+                     }
```

Comment:
`end` at 122, 21 is not aligned with `let(:alter_deck) { FactoryGirl.create :deck, title: "alter_title",` at 120, 4